lang/cs/Org.Apache.REEF.Tang.Tests/Injection/TestMultipleConstructors.cs

```diff
@@ -37,26 +37,26 @@ public class TestMultipleConstructors
         [TestMethod]
         public void TestMissingAllParameters()
         {
-            //Multiple infeasible plans: Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest
-            //, Org.Apache.REEF.Tang.Test
-            //, Version=1.0.0.0
-            //, Culture=neutral
-            //, PublicKeyToken=null:
+            // Multiple infeasible plans: Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest
+            // , Org.Apache.REEF.Tang.Test
+            // , Version=1.0.0.0
+            // , Culture=neutral
+            // , PublicKeyToken=null:
 
-            //  [ Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest = new Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest
-            //    ( System.Boolean Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest+NamedBool = 
-            //    , System.Int32 Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest+NamedInt = 
-            //    ) 
-            //    | new Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest
-            //    ( System.String Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest+NamedString = 
-            //    , System.Int32 Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest+NamedInt = 
-            //    ) 
-            //    | new Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest
-            //    ( System.Boolean Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest+NamedBool = 
-            //    , System.String Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest+NamedString = 
-            //    , System.Int32 Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest+NamedInt = 
-            //    )
-            //  ]
+            ////  [ Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest = new Org.Apache.REEF.Tang.Tests.Injection.MultiConstructorTest
```

Comment:
I mean, all commented out section, starting with `Multiple infeasible plans` and ending with `]`, looks like part of the same message, not two separate messages. So it all should use either // or ////, but not different styles for different parts of the message